src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder();
+        Plane firstPlane = new Plane.PlaneBuilder()
+                .setName("Boeing")
+                .setModel("737-800")
+                .setColour("Green")
+                .setYear(1996)
+                .setPassengerCapacity(198)
+                .build();
+
+        Plane secondPlane = new Plane.PlaneBuilder()
+                .setName("Airbus")
+                .setModel("a380")
+                .build();
+
+        System.out.println(firstPlane.toString());
```

Comment:
```suggestion
        System.out.println(firstPlane);
```
you can write this way as well

Reply:
done